### DISCUSSION TITLE
Redirect auto url post

### DISCUSSION THREAD

**Discussion by elgrosboub** [SiroDiaz/laravel-redirection#3] [Q&A]
Hello,

Is it possible to automatically redirect a url POST to a new one without register all urls manually ?

Example :

test.com/category/url-of-the-post/ => test.com/category/category2/url-of-the-post/

Thx

**Comment by SiroDiaz** [comment]
Hello,

Yes, you can redirect your url post without register all urls manually. Just use the "database" driver, run the migration for this package and finally you can do CRUD operations programatically with your redirections.
